content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -218,6 +225,11 @@ equal to one. The `nginx` container is granted 2 exclusive CPUs.
 
 #### Static policy options
 
+You can toggle on/off groups of options by their maturity level using the following feature gates:
```

Comment:
nit: "You can toggle groups of options on and off based upon their maturity level..."

Reply:
thanks, fixed